### PR TITLE
Finish ECS migration for Packetbeat

### PR DESCRIPTION
**Pull request by andrewkroh**
NOTE: This is based on some other open PRs and will be rebased after they merge. Please only review the last commit.

This should finish out the field updates for Packetbeat.

- Assert that common ECS fields exist in system tests
- Add aliases
- Clean up config files
- Remove unused fields from fields.common.yml
- Remove time_zone from dashboards
- Update navigation links on dashboards to include all dashboards
- Add DNS / TLS to overview dashboard to highlight the capabilities
- Update fields used in the documentation
- Move RPC fields to NFS from Mongo (they were in the wrong package)

Part of #7968

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -70,12 +70,14 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
...
 - Changed AMQP protocol fields to align with ECS. {pull}10090[10090]
 - Changed Redis protocol fields to align with ECS. {pull}10126[10126]
+- Changed pgsql protocol fields to align with ECS. {pull}10147[10147]
 - Changed HTTP protocol fields to align with ECS. {pull}9976[9976]
 - Changed MongoDB protocol fields to align with ECS. {pull}10158[10158]
 - Changed MySQL protocol fields to align with ECS. {pull}10155[10155]
 - Changed NFS protocol fields to align with ECS. {pull}10153[10153]
 - Changed Thrift protocol fields to align with ECS. {pull}10125[10125]
 - Changed Cassandra protocol fields to align with ECS. {pull}10093[10093]
+- Changed Memcache protocol fields to align with ECS. {pull}10189[10189]
```
[the remark this answers is not in the document]
Comment (andrewkroh) [reply CHANGELOG.next.asciidoc]: Correct. I think you missed the note at the top of the PR description.

> NOTE: This is based on some other open PRs and will be rebased after they merge. Please only review the last commit.

[the remark this answers is not in the document]
Comment (webmat) [reply CHANGELOG.next.asciidoc]: You're right, sorry. All of my other comments are about the most recent commit, however. I'll resolve this thread.

File: packetbeat/_meta/fields.common.yml [hunk 2]
```diff
@@ -203,61 +212,19 @@
...
-
-    - name: loadtime
-      type: long
-      description: >
-        The time it takes for the content to be loaded. This is typically
-        used for RUM (real-user-monitoring) but it can make sense in other
-        cases as well.
-        The precision is in microseconds.
-
-    - name: domloadtime
```
[the remark this answers is not in the document]
Comment (andrewkroh) [reply packetbeat/_meta/fields.common.yml]: Correct. None of these were ever used by Packetbeat.

File: packetbeat/docs/packetbeat-filtering.asciidoc [hunk 3]
```diff
@@ -3,18 +3,18 @@
...
   - include_fields:
       fields:
-        - bytes_in
-        - bytes_out
-        - ip
-        - client_ip
+        - client.bytes
+        - server.bytes
+        - client.ip
+        - server.ip
```
[the remark this answers is not in the document]
Comment (andrewkroh) [reply packetbeat/docs/packetbeat-filtering.asciidoc]: All protocols populate client/server and source/destination. I don't think there's much of a need to mention source/destination.

File: dev-tools/ecs-migration.yml [hunk 4]
```diff
@@ -826,30 +826,28 @@
...
-  comment: Don't add an alias until all of Packetbeat stops using this field.
+  alias: true
 
 - from: responsetime
   to: event.duration
   alias: false
-  comment: >
-    Units changed from usec to nsec. Don't add an alias until all of Packetbeat
-    stops using this field.
+  comment: The units changed so no alias was added.
```
[the remark this answers is not in the document]
Comment (andrewkroh) [reply dev-tools/ecs-migration.yml]: I added the `scale` field for `responsetime`.

**Comment by andrewkroh** [comment]
Rebased. The pgsql commits are removed.

**Comment by webmat** [comment]
Since this is a cleanup PR after all the other ones, what's the changelog customs? Add a mention of this PR on each changelog entry? Or are the current entries enough?

**Comment by andrewkroh** [comment]
@webmat I have consolidated the changelog entries into one that points to the overall issue. The issue has links to each PR.

File: CHANGELOG.next.asciidoc [hunk 5]
```diff
@@ -75,21 +75,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Adjust Packetbeat `http` fields to ECS Beta 2 {pull}9645[9645]
   - `http.request.body` moves to `http.request.body.content`
   - `http.response.body` moves to `http.response.body.content`
-- Changed DNS protocol fields to align with ECS. {pull}9941[9941]
+- Changed Packetbeat fields to allign with ECS. {issue}7968[7968]
```
Comment (webmat) [inline CHANGELOG.next.asciidoc]: align

**Comment by andrewkroh** [comment]
> If I understand correctly, method stays there for non-HTTP protocols that still have a "method", and path stays for paths that aren't URLs?

There are a few protocols populating `method` (amqp, pgsql, thrift, redis, mysql, memcache, dns). It probably makes sense for http to continue populating `method` as well. Will see about doing that.

File: CHANGELOG.next.asciidoc (hunk 5 above)
Comment (andrewkroh) [reply CHANGELOG.next.asciidoc]: Fixed. Thanks.

**Comment by webmat** [comment]
Agreed on always filling `method`

**Pull request merged**